plugins/00_dokku-standard/commands

```diff
@@ -193,8 +193,18 @@ case "$1" in
       echo "$SCHEME://$(< "$DOKKU_ROOT/VHOST")"
     else
       for PORT_FILE in $DOKKU_ROOT/$APP/PORT.*; do
-        echo "$SCHEME://$(< "$DOKKU_ROOT/HOSTNAME"):$(< "$PORT_FILE")"
+        echo "http://$(< "$DOKKU_ROOT/HOSTNAME"):$(< "$PORT_FILE") (container)"
```

Comment:
this is only if vhost is disabled and we print out the ssl version only if we have a port